utils/docker/Dockerfile.leap.15

```diff
@@ -13,17 +13,30 @@ LABEL maintainer="daos@daos.groups.io"
 ARG CB0
 
 # Use our own repo cache
-ARG REPO_URL=""
-ARG REPO_GROUP_LEAP15=""
-RUN if [ -n "${REPO_GROUP_LEAP15}" ]; then				\
-    rm -f /etc/zypp/repos.d/*.repo &&					\
-    zypper --non-interactive ar ${REPO_URL}${REPO_GROUP_LEAP15} daos-stack-group-repo && \
-    zypper --non-interactive mr --gpgcheck-allow-unsigned-repo daos-stack-group-repo; \
-  fi
 
-# Make sure repos are up to date
-RUN zypper --non-interactive --no-gpg-checks --gpg-auto-import-keys refresh
+# If a DISTRO repo is provided then use it.
+ARG REPO_URL=""
+ARG REPO_DISTRO=""
+RUN if [ -n "${REPO_DISTRO}" ]; then				                           \
```

Comment:
If pushing again it would be worth checking spaces vs tabs for this block.

Reply:
Looks like I missed some with my last pass through it, including it with the typo fix.